packages/appstore/src/types/params.types.ts

```diff
@@ -1,5 +1,5 @@
 export type TRoute = {
-    component?: React.ReactNode;
+    component?: () => void;
```

Comment:
Type '() => JSX.Element' is not assignable to type 'ReactNode'.ts(2322)

Reply:
I had an issue **Type '() => JSX.Element' is not assignable to type 'ReactNode'.ts(2322)** in **src/constants/routes-config.ts file** (for **TradingHub** component) which failed my build process (after installing @types/react package). one way to fix that was changing the type of the component. the reason I changed the type was that. now I see it would be better to write it like this **() => JSX.Element | React.ReactNode**

Reply:
Since react components are functions that return something we need to change the type to a function type which return reactnode (as matin said to support a wider range of types).
Making this small change fixes the issue that we have in our build process.